### PR TITLE
Adaugat animatii si blur la Cosplay si Partener Media

### PR DESCRIPTION
**Pull request by smiromania**
nu sunt ciordite cu siguranta acele animatii de la componente

### CODE REVIEW

File: client/src/components/Cosplay.js [hunk 1]
```diff
@@ -16,28 +17,64 @@ function SignupButton() {
   );
 }
```
Comment (cristim67) [inline client/src/components/Cosplay.js:18]: You should delete them

File: client/src/components/Cosplay.js [hunk 2]
```diff
@@ -16,28 +17,64 @@ function SignupButton() {
...
+          id="projects"
+          variants={{
+            hidden: { opacity: 0, y: 75 },
+            visible: { opacity: 1, y: 0 },
+          }}
+          initial="hidden"
+          animate={mainControls}
+          transition={{ duration: 0.5, delay: 0.25 }}
+        >
+          {/* <center className="text-3xl font-bold text-[#78e800]">COSPLAY</center> */}
```
Comment (cristim67) [inline client/src/components/Cosplay.js]: comments with an unused code should disappear

File: client/src/components/Cosplay.js [hunk 3]
```diff
@@ -16,28 +17,64 @@ function SignupButton() {
...
+              <div className="pointer-events-none absolute top-0 left-0 w-full h-full flex flex-col items-center justify-center">
+                <span
+                  style={{
+                    textShadow: '-2px 0 black, 0 2px black, 2px 0 black, 0 -2px black'
+                }}
+                  className="text-5xl pb-16 text-stroke-3 font-bold text-[#78e800] mb-4"
+                >
+                  COSPLAY
+                </span>{" "}
+                {/* Adăugat aici */}
```
Comment (cristim67) [inline client/src/components/Cosplay.js]: comments with an unused code should disappear and "{ " " }"

File: client/src/components/Cosplay.js [hunk 4]
```diff
@@ -16,28 +17,64 @@ function SignupButton() {
...
+                <span className="md:text-3xl text-xs text-white hover:duration-300 hover:bg-opacity-0 bg-black px-2 py-1 rounded">
+                  Competiția cosplay la LanParty LSE: tehnologia întâlnește arta
+                  și pasiunea.
+                </span>
+              </div>
+            </div>
+            <div className="text-center pt-16 pb-32">{SignupButton()}</div>
```
Comment (cristim67) [inline client/src/components/Cosplay.js:69]: this line (70) is not used

File: client/src/components/PartenerMedia.js [hunk 5]
```diff
@@ -82,53 +84,69 @@ function YouTubeButton(text1, text2) {
...
+        animate={mainControls}
+        transition={{ duration: 0.5, delay: 0.25 }}
+      >
+        <center className="pb-16">
+          <h1 className="text-3xl font-bold text-white">
+            ALLSTARS GAMING: Partener Media
+          </h1>
+        </center>
+        <center className="pb-16 transform transition-transform duration-600 hover:scale-110">
+          <a href="https://allstars-gaming.ro/">
```
Comment (cristim67) [inline client/src/components/PartenerMedia.js]: try not to center with the center tag. here is an example : https://tailwindcss.com/docs/align-content

Comment (cristim67) [inline client/src/components/PartenerMedia.js]: you should use target="_blank" for links

File: client/src/components/PartenerMedia.js [hunk 6]
```diff
@@ -82,53 +84,69 @@ function YouTubeButton(text1, text2) {
...
+            <img
+              src="https://allstars-gaming.ro/wp-content/uploads/2022/07/logo-allstars-mic2.png"
+              alt=""
```
Comment (cristim67) [inline client/src/components/PartenerMedia.js]: the images should have another attribute. https://www.w3schools.com/tags/att_img_alt.asp

**Review by cristim67 — CHANGES_REQUESTED** [review]
use priettier for identification

File: client/src/components/Cosplay.js [hunk 7]
```diff
@@ -1,4 +1,5 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
+import { motion, useInView, useAnimation } from "framer-motion";
 import Poza from "../assets/img/Colaj.png";
```
Comment (cristim67) [inline client/src/components/Cosplay.js]: images should be .webp

Comment (cristim67) [reply client/src/components/Cosplay.js]: change the name of the variable to a more suggestive name such as Colaj

File: client/src/components/PartenerMedia.js [hunk 8]
```diff
@@ -1,4 +1,6 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
+import { motion, useInView, useAnimation } from "framer-motion";
+
 function TikTokButton(text1, text2) {
   const totalLength = text1.length + text2.length;
   const ratio1 = text1.length / totalLength;
```
Comment (cristim67) [inline client/src/components/PartenerMedia.js]: change the name of the variable to a more suggestive name

File: client/src/components/Cosplay.js (hunk 4 above)
Comment (smiromania) [reply client/src/components/Cosplay.js:69]: ???

File: client/src/components/PartenerMedia.js (hunk 8 above)
Comment (smiromania) [reply client/src/components/PartenerMedia.js]: which one?

File: client/src/components/Cosplay.js (hunk 4 above)
Comment (cristim67) [reply client/src/components/Cosplay.js:69]: you are calling a function that is not defined

File: client/src/components/PartenerMedia.js (hunk 8 above)
Comment (cristim67) [reply client/src/components/PartenerMedia.js]: text1, text2, ratio1

File: client/src/components/Cosplay.js [hunk 9]
```diff
@@ -16,28 +17,63 @@ function SignupButton() {
...
-        <div className="text-center pt-16 pb-32">
-          {SignupButton()}
-        </div>
+        </motion.div>
       </div>
     </>
   );
 }
+
 //vrem vreme buna muie lu furtuna ca nu ne-a lasat sa punem animatia
```
Comment (cristim67) [inline client/src/components/Cosplay.js:77]: like this

**Review by cristim67 — APPROVED** [review]
Good job

**Pull request merged**